### PR TITLE
e2e event test

### LINKED ISSUE #1426: Create e2e tests for event api
Dependant on https://github.com/mesg-foundation/engine/issues/1423

Create e2e tests for the event api with a running instance:
- [ ] create (done by instance)
- [ ] stream

For this tests, i suggest the following:
1. starts a stream to listen for event
2. creates an execution for the instance that will actually emit an event
3. waits for an event to pop on the stream
4. checks the data

### PR DESCRIPTION
**Pull request by krhubert**
dep https://github.com/mesg-foundation/engine/pull/1442
close #1426

### CODE REVIEW

File: e2e/event_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		Key:          "ping_ok",
+	})
+	require.NoError(t, err)
+
+	event, err := stream.Recv()
+	require.NoError(t, err)
+
+	require.Equal(t, resp.Hash, event.Hash)
+	require.Equal(t, testInstanceHash, event.InstanceHash)
+	require.Equal(t, "ping_ok", event.Key)
```
Comment (NicolasMahe) [inline e2e/event_test.go]: Can you add a check on the event's data?

Comment (krhubert) [reply e2e/event_test.go]: Can you remind me why have you removed `compare` opinion for gogo.protobuf types?

Comment (NicolasMahe) [reply e2e/event_test.go]: i did? i think we should have those helpers (compare, equal, etc..)

Comment (NicolasMahe) [reply e2e/event_test.go]: actually, only process, ownership and execution have `option (gogoproto.equal_all) = true;`
we can definitely set this on all proto

**Pull request merged**